lib/tasks/seek_upgrades.rake

```diff
@@ -131,4 +132,31 @@ namespace :seek do
       puts "Database adapter is: #{ActiveRecord::Base.connection.instance_values["config"][:adapter]}, doing nothing"
     end
   end
+
+
+  task(update_thesis_related_publication_types: [:environment]) do
+    puts 'Updating publication types ...'
+
+    unless PublicationType.find_by(title:"Masters Thesis").nil?
+      PublicationType.find_by(key:"mastersthesis").update_attributes(title:"Master's Thesis")
```

Comment:
not a blocker, but to make you aware. With Rails 6 (on the master branch), `update_attributes `is deprecated and should instead use just `update`. It doesn't affect 1.12 though, so I can update this when we merge.